at_secondary/at_persistence_secondary_server/lib/src/log/commitlog/commit_log_keystore.dart

```diff
@@ -129,7 +129,14 @@ class CommitLogKeyStore
   @override
   Future<void> remove(int commitId) async {
     try {
+      final commitEntry = (await _getBox() as Box).get(commitId);
       await _getBox().delete(commitId);
+      // invalidate cache for the removed entry
+      if (commitEntry != null) {
+        final key = commitEntry.atKey;
+        final value = _commitLogCacheMap.remove(key);
+        _logger.finest('removed value: $value');
+      }
```

Comment:
We can reduce the temp variables here:

 if (commitEntry != null) {
     
         _commitLogCacheMap.remove(commitEntry.atKey);
        _logger.finest('removed key : $commitEntry.atKey from commit log');
      }

Reply:
+ Log statement can be a bit more descriptive